### PR TITLE
Update valk.json

### PR DESCRIPTION
**Pull request by grtll**
I added this pack's cards text.

### CODE REVIEW

File: pack/valk.json [hunk 1]
```diff
@@ -348,12 +378,14 @@
 		"cost": 0,
 		"deck_limit": 3,
 		"faction_code": "basic",
+		"flavor": "\Showoff.\" —Jane Foster",
```
Comment (Kamalisk) [inline pack/valk.json:381]: missing a quote here

File: pack/valk.json [hunk 2]
```diff
@@ -348,12 +378,14 @@
...
 		"name": "Godllike Stamina",
 		"octgn_id": "dd904456-03ca-4ae7-9755-e3d9f148357c",
 		"pack_code": "valk",
 		"position": 24,
 		"quantity": 3,
 		"resource_physical": 1,
+		"text": "Play onlu if your identity has the <b>ASGARD<b> trait.\n<b>Action<b>: Heal 2 damage from your identity. You may discard a status card from your identity.",
```
Comment (Kamalisk) [inline pack/valk.json:388]: onlu I assume is only

File: pack/valk.json [hunk 3]
```diff
@@ -388,6 +420,7 @@
 		"position": 33,
 		"quantity": 3,
 		"resource_physical": 1,
+		"text": "Alliance.\n<b>Hero Action<b> (thwart): Exhaust an <b>AVENGER<b> character and a <b>GUARDIAN<b> character -> remove X threat from each scheme, where X is equal to the combined THW of those characters.",
```
Comment (Kamalisk) [inline pack/valk.json:423]: closing tags need to be </b> seem to have use <b> on both sides

File: pack/valk.json [hunk 4]
```diff
@@ -340,6 +369,7 @@
 		"position": 23,
 		"quantity": 1,
 		"resource_energy": 1,
+		"text": "Play only if you identity has the <b>ASGARD<b> trait.\n<b>Action<b>: Exhaust The Bifrost -> search your deck for an <b>ASGARD<b> ally and play it (paying its cost). Shuffle your deck.",
```
Comment (Kamalisk) [inline pack/valk.json:372]: For traits, we use [[asgard]] and you don't need to put it in capital letters.

File: pack/valk.json [hunk 5]
```diff
@@ -430,6 +466,7 @@
 		"position": 36,
 		"quantity": 3,
 		"resource_energy": 1,
+		"text": "Alliance. 'The players can pay this card's costs as a group.)\n<b>Hero Action<b>: Choose an <b>AVENGER<b> character and a <b>GUARDIAN<b> character -> ready each of those characters.",
```
Comment (Kamalisk) [inline pack/valk.json:469]: missing a starting (

**Comment by Kamalisk** [comment]
made some comments, mostly good but a few things to fix. 
I should update the readme with more of what is used.

File: pack/valk.json (hunk 3 above)
Comment (grtll) [reply pack/valk.json:423]: if thid comment was about blodness tags, I fixed the tags (sorry not a native english speaker).

Comment (Kamalisk) [reply pack/valk.json]: well whatever I wrote didnt make much sense, I did mean to say closing tags need to be </b>

**Comment by Kamalisk** [comment]
looks great now as far as I can see. Will merge and update.
Thanks a lot for helping.

**Pull request merged**